src/renderer/views/Subscriptions/Subscriptions.js

```diff
@@ -1,492 +1,45 @@
 import { defineComponent } from 'vue'
-import { mapActions, mapMutations } from 'vuex'
-import FtLoader from '../../components/ft-loader/ft-loader.vue'
+
+import SubscriptionsVideos from '../../components/subscriptions-videos/subscriptions-videos.vue'
+import SubscriptionsLive from '../../components/subscriptions-live/subscriptions-live.vue'
+import SubscriptionsShorts from '../../components/subscriptions-shorts/subscriptions-shorts.vue'
+
 import FtCard from '../../components/ft-card/ft-card.vue'
-import FtButton from '../../components/ft-button/ft-button.vue'
-import FtIconButton from '../../components/ft-icon-button/ft-icon-button.vue'
 import FtFlexBox from '../../components/ft-flex-box/ft-flex-box.vue'
-import FtElementList from '../../components/ft-element-list/ft-element-list.vue'
-import FtChannelBubble from '../../components/ft-channel-bubble/ft-channel-bubble.vue'
-
-import { calculatePublishedDate, copyToClipboard, showToast } from '../../helpers/utils'
-import { invidiousAPICall } from '../../helpers/api/invidious'
-import { getLocalChannelVideos } from '../../helpers/api/local'
 
 export default defineComponent({
   name: 'Subscriptions',
   components: {
-    'ft-loader': FtLoader,
+    'subscriptions-videos': SubscriptionsVideos,
+    'subscriptions-live': SubscriptionsLive,
+    'subscriptions-shorts': SubscriptionsShorts,
     'ft-card': FtCard,
-    'ft-button': FtButton,
-    'ft-icon-button': FtIconButton,
-    'ft-flex-box': FtFlexBox,
-    'ft-element-list': FtElementList,
-    'ft-channel-bubble': FtChannelBubble
+    'ft-flex-box': FtFlexBox
   },
   data: function () {
     return {
-      isLoading: false,
-      dataLimit: 100,
-      videoList: [],
-      errorChannels: [],
-      attemptedFetch: false,
+      currentTab: 'videos'
     }
   },
```

Comment:
Add remembering last tab
Quite common to browse a video (live/short) and then go back to feed
```suggestion
  },
  watch: {
    currentTab(value) {
      // Save last used tab, restore when view mounted again
      sessionStorage.setItem('Subscriptions/currentTab', value)
    },
  },
  mounted: async function () {
    // Restore currentTab
    const lastCurrentTabId = sessionStorage.getItem('Subscriptions/currentTab')
    if (lastCurrentTabId != null) { this.changeTab(lastCurrentTabId) }
  },
```